src/app/views/discover/games/list/list.ts

```diff
@@ -86,7 +88,31 @@ export default class RouteDiscoverGamesList extends BaseRouteComponent {
 	}
 
 	get routeTitle() {
+		let title = this.listTitle;
```

Comment:
I think we should make a private class that takes what's needed to generate this title in the constructor and then has a "title" getter. It's just so much code, and we have all these random helper getters to generate it. What do you think?

Reply:
I feel like Title code should be centralized somewhere, otherwise it'd be hell if we want to change branding some day.
Have functions like getCommunityTitle and pass in args like the community name.

Reply:
I don't want to get into this sort of refactoring at this stage. Remember this was meant to be a single day thing. Theres a lot to be desired for sure but i'd like to dedicate more time to do it right when i revisit this.